controller/rest/system.go

```diff
@@ -2116,6 +2117,223 @@ func handlerSystemGetRBAC(w http.ResponseWriter, r *http.Request, ps httprouter.
 	restRespSuccess(w, r, &resp, acc, login, nil, "Get missing Kubernetes RBAC")
 }
 
+func getAlertGroup(alerts []string, alertType api.AlertType, acceptedAlerts utils.Set) (*api.RESTNvAlertGroup, error) {
+	alertGroup := &api.RESTNvAlertGroup{
+		Type: alertType,
+	}
+
+	if len(alerts) > 0 {
+		for _, alert := range alerts {
+			b := md5.Sum([]byte(alert))
+			key := hex.EncodeToString(b[:])
+			if !acceptedAlerts.Contains(key) {
+				alertGroup.Data = append(alertGroup.Data, &api.RESTNvAlert{
+					ID:      key,
+					Message: alert,
+				})
+			}
+		}
+	}
+
+	if len(alertGroup.Data) > 0 {
+		return alertGroup, nil
+	}
+
+	return alertGroup, errors.New("no alert is retrieved")
+}
+
+func getInternalCertExpireAlert(certFilePath string) (string, error) {
+	expireThresholds := []time.Duration{30 * 24 * time.Hour, 90 * 24 * time.Hour, 180 * 24 * time.Hour}
```

Comment:
minor: How about using #day instead of time.Duration, so we can format the alert directly in the switch case below?  For example:

```
expireThreholdDays := []int{30, 90, 180}
...
for _, expireThresholdDay := range expireThreholdDays {
    certExpiredMsg = fmt.Sprintf("Internal certificate will be expired in %d day(s)", expireThresholdDay)
}
```

Reply:
changed